### PR TITLE
Add package to define custom trailbot msgs and srv

### PR DESCRIPTION
**Pull request by barzanisar**

### CODE REVIEW

File: trailbot_interfaces/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# trailbot_interfaces
+This package will have contain custom Trailbot services and msgs.
+
+## Make include src 
+```bash
+cd ~/ros2_ws/src/TRAILBot/trailbot_interfaces
+mkdir include src msg
```
Comment (jacobdeery) [inline trailbot_interfaces/README.md:7]: What is this meant to do?

File: trailbot_interfaces/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (jacobdeery) [inline trailbot_interfaces/LICENSE:1]: Let's just keep one license file at the top level

**Review by jacobdeery — APPROVED** [review]
LGTM, just two minor comments

File: trailbot_interfaces/README.md (hunk 1 above)
Comment (barzanisar) [reply trailbot_interfaces/README.md:7]: 'include' and 'src' folders is what the package creates automatically, so I initially thought building the pkg requires these folders. I tried building the pkg without them and it builds so removing this line.

Comment (barzanisar) [reply trailbot_interfaces/README.md]: I removed this line

File: trailbot_interfaces/LICENSE (hunk 2 above)
Comment (barzanisar) [reply trailbot_interfaces/LICENSE]: done!

**Pull request merged**